CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-beta6 (2020-09-14)
+
+**For beta releases**, upgrading from an earlier version of the
+extension (including previous beta releases) is not supported.
+
+This beta release includes breaking changes to APIs. The most 
+notable changes are:
```

Comment:
```suggestion
notable changes since the beta-5 release are the following, which will
be reflected in forthcoming documentation for the 2.0 release.
```

Reply:
added